libs/blocks/card/card.js

```diff
@@ -17,24 +17,40 @@ const getCardType = (styles) => {
   return cardTypes[authoredType] || HALF;
 };
 
+const getUpFromSectionMetadata = (section) => {
+  const sectionMetadata = getSectionMetadata(section.querySelector('.section-metadata'));
+  const styles = sectionMetadata.style?.split(', ').map((style) => style.replaceAll(' ', '-'));
+  return styles?.find((style) => style.includes('-up'));
+};
+
 const addWrapper = (el, section, cardType) => {
-  const { classList } = section;
   const gridCl = 'consonant-CardsGrid';
+  const prevGrid = section.querySelector(`.consonant-Wrapper .${gridCl}`);
 
-  if (!classList.contains('consonant-Wrapper')) {
-    const up = Array.from(classList).find((style) => style.includes('-up'))?.replace('-', '') || '3up';
-    const innerWrapper = `<div class="consonant-Wrapper-inner">
-      <div class="consonant-Wrapper-collection">
-        <div class="${gridCl} ${gridCl}--${up} ${gridCl}--with4xGutter${cardType === DOUBLE_WIDE ? ` ${gridCl}--doubleWideCards` : ''}">
-        </div>
-      </div>
-    </div>`;
-
-    classList.add('consonant-Wrapper', 'consonant-Wrapper--1200MaxWidth');
-    section.insertAdjacentHTML('afterbegin', innerWrapper);
+  if (prevGrid) {
+    prevGrid.append(el);
+    return;
   }
 
-  section.querySelector(`.${gridCl}`).append(el);
+  const upClass = getUpFromSectionMetadata(section);
+  const up = upClass?.replace('-', '') || '3up';
+
+  const prevSib = el.previousElementSibling;
+  const nextSib = el.nextElementSibling;
```

Comment:
supernit: move this to after `const wrapper` since it's not used till after that

Reply:
Actually, this needs to stay here. `el` gets moved on line 41 so its siblings change.